redpanda/templates/configmap.yaml

```diff
@@ -91,7 +96,20 @@ data:
           port: {{ $values.listeners.rpc.port }}
 {{- end }}
     rpk:
-{{ toYaml .Values.rpk | indent 6 }}
+      enable_usage_stats: {{ .Values.logging.usageStats.enabled }}
+{{- if hasKey .Values.resources.cpu "overprovisioned" }}
+      overprovisioned: {{ .Values.resources.cpu.overprovisioned }}
+{{- else }}
+      overprovisioned: false
+{{- end }}
+{{- if hasKey .Values.resources.memory "enable_memory_locking" }}
+      enable_memory_locking: {{ .Values.resources.memory.enable_memory_locking }}
+{{- else }}
+      enable_memory_locking: false
+{{- end }}
+      tune_cpu: false
+      tune_disk_irq: false
```

Comment:
I think we can delete this , rpk will strip false values anyway (https://github.com/redpanda-data/redpanda/commit/81b6236ec06d7554caec841fe1a89df8bdf1be00), or do we keep them here so the users know that this can't be set as true?

Reply:
Ahh interesting, I didn't realize they were stripped out if `false`. I added them here just because I wasn't sure of the default value, and I knew they needed to always be false when running in a container. If the default for these values are false, then these can be removed.

Another layer we could have is to ensure these parameters are ignored if the user adds them in manually... a long shot but possible given these details can be found throughout our docs.

Reply:
I see the documentation lists these two parameters as `false` by default, so I'll remove them completely.

Reply:
Agree with the layer to ignore these parameters, if it's outside of the scope of this PR maybe a comment in `Values.yaml` could be useful.

Reply:
Both (comments in `values.yaml` and ignoring these if manually added) are easy to do and I'll add them in when removing these parameters from `configmap.yaml`.

Reply:
Made this change here: https://github.com/redpanda-data/helm-charts/pull/127/commits/1f9af5277b045b0c6c02376f46290f739f88723c

Reply:
Awesome, thanks!